.eslintrc.js

```diff
@@ -55,33 +49,48 @@ module.exports = {
       files: ["src/languages/*.js"],
       rules: {
         "no-unused-expressions": "off",
-        // // languages are all over the map and we don't want to
-        // // do a mass edit so turn off the most egregious rule violations
+        // languages are all over the map and we don't want to
+        // do a mass edit so turn off the most egregious rule violations
         // indent: "off",
-        camelcase: 0,
-        "no-control-regex": 0,
-        "no-useless-escape": 0,
-        "comma-dangle": 1,
+        camelcase: "off",
+        "no-control-regex": "off",
+        "no-useless-escape": "off",
+        "comma-dangle": "warn",
         "array-bracket-spacing": ["error", "always"
-        //   {
-        //     objectsInArrays: true
+          // {
+          //   objectsInArrays: true
           // }
         ],
-        // "object-curly-spacing": 1,
+        // "object-curly-spacing": "warn",
         // "key-spacing": "off",
-        // "array-bracket-spacing": [1],
-        "array-bracket-newline": [1, {
+        // "array-bracket-spacing": ["warn"],
+        "array-bracket-newline": ["warn", {
           multiline: true,
           minItems: 2
         }],
-        "array-element-newline": 1,
+        "array-element-newline": "warn",
         "object-curly-newline": [1, {
           minProperties: 1
         }],
         "object-property-newline": [2,
           { allowAllPropertiesOnSameLine: false }
         ]
       }
+    },
+    {
+      files: ["test/**/*.js"],
+      env: {
+        mocha: true
+      },
+      parserOptions: {
+        ecmaVersion: 2021
```

Comment:
Does Node 10 support all 2021 ECMA stuff?  Technically we only test against 12+ but 10 is still a supported release and as far as I know we work on 10 (we should perhaps CI it as well)... so since this code should work on Node 10 we need to set ecmaVersion accordingly. Thoughts?

Reply:
I'd have approved 2018 without any thought but 2021 is a bit "fresh" so I wonder...... :-)

Reply:
I don't actually know. I'm fine setting it to `"ecmaVersion": 2018` which should cover Node 10 (released in 2018), but wouldn't suggest going lower to not get errors on `async` for example.

There is an ESLint plugin (`eslint-plugin-node`) that offers a rule (`node/no-unsupported-features`) to test for supported/unsupported node features directly, but that feels a bit like overkill since this is only the test code.

Reply:
Oh that might be nice for the browser though, I'll have to check it out.  Lets do 2018 then if you don't know.

Reply:
Done.